NorthstarDLL/squirrelnativeautobind.h

```diff
@@ -0,0 +1,97 @@
+#pragma once
+#include <vector>
+
+typedef void (*SqAutoBindFunc)(void);
+
+class SquirrelAutoBindContainer
+{
+  public:
+	std::vector<SqAutoBindFunc> clientSqAutoBindFuncs;
+	std::vector<SqAutoBindFunc> serverSqAutoBindFuncs;
+
+	SquirrelAutoBindContainer()
+	{
+		clientSqAutoBindFuncs = {};
```

Comment:
don't think these should need manual initialisation in this case? could just initialise them in the property definition if needed anyway, manually defining a constructor isn't necessary